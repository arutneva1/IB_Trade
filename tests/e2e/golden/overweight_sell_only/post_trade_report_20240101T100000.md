| symbol | side | filled_shares | avg_price | notional | avg_slippage | residual_drift_bps |
| --- | --- | --- | --- | --- | --- | --- |
| BBB | BUY | 11.0000 | 51.00 | 561.00 | 0.00 | 1.41 |
| AAA | BUY | 43.0000 | 101.00 | 4343.00 | 0.00 | 82.69 |
