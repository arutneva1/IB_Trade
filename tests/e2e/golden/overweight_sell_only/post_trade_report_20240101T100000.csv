symbol,side,filled_shares,avg_price,notional,avg_slippage,residual_drift_bps
BBB,BUY,11.0,51.0,561.0,0.0,1.41
AAA,BUY,43.0,101.0,4343.0,0.0,82.69
